src/Rector/AbstractPHPUnitRector.php

```diff
@@ -2,21 +2,30 @@
 
 namespace Rector\Rector;
 
-use Nette\Utils\Strings;
 use PhpParser\Node;
-use Rector\Node\Attribute;
+use PHPUnit\Framework\TestCase;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
 
 abstract class AbstractPHPUnitRector extends AbstractRector
 {
     /**
-     * Check if the file is a PHPUnit TestCase. By default, it should end with "Test", as it is the standard.
-     *
-     * @see https://phpunit.de/getting-started-with-phpunit.html
+     * @var MethodCallAnalyzer
      */
+    private $methodCallAnalyzer;
+
+    public function __construct(MethodCallAnalyzer $methodCallAnalyzer)
```

Comment:
@TomasVotruba Should we revert 691eb10? Because doing this way, as the classes that extends it, will overwrite this method :(